src/Microsoft.VisualBasic/tests/Microsoft/VisualBasic/CompilerServices/StructUtilsTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.VisualBasic.CompilerServices.Tests
+{
+    public static class StructUtilsTestData
+    {
+        public static TheoryData<object, int> RecordsAndLength() => new TheoryData<object, int>
+        {
+            { null, 0 },
+            { new Struct_Empty(), 0 },
+            { new Struct_T<float>(), 4 },
+            { new Struct_T<double>(), 8 },
+            { new Struct_T<short>(), 2 },
+            { new Struct_T<int>(), 4 },
+            { new Struct_T<byte>(), 1 },
+            { new Struct_T<long>(), 8 },
+            { new Struct_T<DateTime>(), 8 },
+            { new Struct_T<bool>(), 2 },
+            { new Struct_T<decimal>(), 16 },
+            { new Struct_T<char>(), 2 },
+            { new Struct_T<string>(), 4 },
+            { new Struct_ArrayT<byte>(elementCount: 10), 4 },
+            { new Struct_ArrayT<int>(elementCount: 10), 4 },
+            { new Struct_FixedArrayT10<byte>(), 11 },
+            { new Struct_FixedArrayT10<int>(), 44 },
+            { new Struct_FixedArrayT11To20<byte>(), 252 }, // Bug?
```

Comment:
>// Bug? [](start = 59, length = 7)

It looks like `VBFixedArray(11, 20)` creates an array of (11+1) * (20+1) elements.

Reply:
(If the tests pass on the desktop leg then this is proven)

Reply:
It is "bug for bug compatible with desktop". Nothing has changed in the code itself. I added comments where I thought I found bugs.
I expected `VBFixedArray(11, 20)` to create an array of 10 elements with lower bound 11 and upper bound 20. Hence the naming of the structs. I was wrong. It creates indeed an array of (11+1)*(20+1) elements.
Tests have been corrected.